fix sparse-checkout list

JamesWTruher opened the pull request:
Run powershell.exe in OpenCover since it will be in the path.
If there's an error in Start-CodeCoverageRun be sure to log as much as possible

JamesWTruher committed:
fix sparse-checkout list

Also run powershell.exe in OpenCover since it will be in the path.
If there's an error in Start-CodeCoverageRun be sure to log as much as possible

adityapatwardhan commented on test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1:
```diff
@@ -275,7 +278,7 @@ try
         Write-LogPassThru -Message "pulling sparse repo"
         "src" | Out-File -Encoding ascii .git\info\sparse-checkout -Force
-        "assets" | Out-File -Encoding ascii .git\info\sparse-checkout -Append
+        "/assets" | Out-File -Encoding ascii .git\info\sparse-checkout -Append
```
Can you add a comment why '/' is required for assets and not for src.

JamesWTruher replied:
i added it for /src as that seems to work and is more clear

JamesWTruher committed:
Fix typo in asset dir removal

and make the directory removal code common
have the sparse checkout include /src instead of just 'src'
